src/components/Explore/interfaces.ts

```diff
@@ -29,24 +29,19 @@ export interface Series {
  * we need to update these assigned indices to match.
  */
 export enum ExploreMetric {
-  WEEKLY_CASES_PER_100K = 0,
-  HOSPITALIZATIONS = 1,
-  ICU_HOSPITALIZATIONS = 2,
-  VACCINATIONS_FIRST_DOSE = 3,
-  VACCINATIONS_COMPLETED = 4,
-  VACCINATIONS_ADDITIONAL_DOSE = 5,
-  ICU_USED = 6,
-  POSITIVITY_RATE = 7,
-  ADMISSIONS_PER_100K = 8,
-  RATIO_BEDS_WITH_COVID = 9,
-
-  /**
-   * Cases and Deaths no longer used in Trends
-   * (i.e. no longer included in EXPLORE_METRICS in Explore/utils.ts),
-   * and so we order them last in the enum.
-   */
-  CASES = 10,
-  DEATHS = 11,
+  ADMISSIONS_PER_100K = 0,
+  RATIO_BEDS_WITH_COVID = 1,
+  WEEKLY_CASES_PER_100K = 2,
+  WEEKLY_DEATHS_PER_100K = 3,
+  CASES = 4,
+  DEATHS = 5,
+  HOSPITALIZATIONS = 6,
+  ICU_HOSPITALIZATIONS = 7,
+  VACCINATIONS_FIRST_DOSE = 8,
+  VACCINATIONS_COMPLETED = 9,
+  VACCINATIONS_ADDITIONAL_DOSE = 10,
+  ICU_USED = 11,
+  POSITIVITY_RATE = 12,
```

Comment:
As per the content team, we're keeping the ordering of existing metrics as is, but putting all new metrics at the top and arranged in alphabetical order.

Reply:
FWIW- It's not worth messing with in this PR, but it seems unnecessarily fragile that we have to keep these assigned IDs in sync with the EXPLORE_METRICS list.  Seems like that's probably avoidable...

Reply:
im 99.99% sure this is my doing/my fault 😭😭😭 . i wired up the enum values to the corresponding item's index in the dropdown menu. its def way too fragile. ill revisit it

Reply:
All good. Not a high priority. 😁